iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepLexicalConstant.java

```diff
@@ -43,6 +43,8 @@ public class BicepLexicalConstant {
   public static final String QUOTED_STRING_LITERAL_NO_QUOTES = "(?:(?!'|\\$\\{)[\\s\\S])*+";
   public static final String IDENTIFIER_LITERAL = "[a-zA-Z_][a-zA-Z_0-9]*+";
   public static final String AMBIENT_TYPE = "(?:array|bool|int|object|string)(?![\\S])";
+  public static final String UNARY_OPERATOR = "!|-|\\+";
+  public static final String MULTILINE_STRING = "(?:(?!''')(?:[\\s\\S])?+)*+";
```

Comment:
I feel like having `BicepLexicalGrammar.MULTILINE_STRING` and `BicepLexicalConstant.MULTILINE_STRING` share the same name is a bit confusing. (Because a `BicepLexicalConstant.MULTILINE_STRING` is part of a `BicepLexicalGrammar.MULTILINE_STRING_REGEX` which is part of a `BicepLexicalGrammar.MULTILINE_STRING`)


 I would suggest to renaming something.
Maybe  `BicepLexicalConstant.MULTILINE_STRING` to  `BicepLexicalConstant.MULTILINE_STRING_CONTENT`